src/olympia/addons/tests/test_admin.py

```diff
@@ -96,6 +96,12 @@ def setUp(self):
         self.admin_home_url = reverse('admin:index')
         self.list_url = reverse('admin:addons_addon_changelist')
 
+    def _call_related_content_method(self, method, addon):
```

Comment:
Is this is only used in one place?  It seems overwritten for that (you wouldn't need the `getattr` stuff either)

Reply:
IMO, we should either generalize that function and move it to `TestCase` (or just in `amo/tests/__init__.py`) so it'd be `def _call_admin_related_content_method(self, admin, method, instance)` (we can get which model with `admin.model`).  Or (easier) just put the functionality in just this class directly into `test_activity`.